packages/connect/src/schemas/paged_request_opts_schema.ts

```diff
@@ -0,0 +1,8 @@
+export const pagedRequestOptsSchema = {
+    id: '/PagedRequestOpts',
+    type: 'object',
+    properties: {
+        page: { type: 'number' },
+        per_page: { type: 'number' },
```

Comment:
This needs to be `perPage`. This is the downside discussed re: allowing arbitrary additional properties in JS objects validated with schemas :(

Reply:
damn nice catch